src/Microsoft.Health.Dicom.Core/Features/Retrieve/IFramesRangeCache.cs

```diff
@@ -15,7 +15,7 @@ public interface IFramesRangeCache
 {
     public Task<IReadOnlyDictionary<int, FrameRange>> GetAsync(
        object key,
-       VersionedInstanceIdentifier input,
-       Func<VersionedInstanceIdentifier, CancellationToken, Task<IReadOnlyDictionary<int, FrameRange>>> asyncFactory,
+       long input,
```

Comment:
Maybe rename this to `watermark` or `version` so we at least know what this `long` is lol

Reply:
Done